swift/swift.py

```diff
@@ -171,13 +175,44 @@ def destroyApp(self, name: str):
     def _broadcast(self, channelName: str, msg: str):
         """Broadcasts the message to the subscriber apps of the channel.
 
+        If channelName is "swift", the message is for system call.
+
         Args:
             channelName: Target channel name.
             msg: Message to be broadcast.
         """
+        if channelName == "swift":
+            self._call_system(msg)
         for app in self._subscribers[channelName]:
             app.received.emit(channelName, msg)
 
+    def _call_system(self, msg: str):
+        """Handles the system call.
+
+        Args:
+            msg: A JSON string of a message about a system call.
+              Its keys represent an action. 
+              All requested actions are performed sequentially.
+              Possible actions are as follows.

```

Comment:
Your idea looks good!
In my method, however, a value of the dictionary is a list so that it can allow duplicate actions.

For example,
```python
msg = {
    "destroy": ["logger", ...],
    "create": [{...}, {...}, ...]
}
```

Then, which do you think better?

Reply:
Let's change this to support only a single command at a time, using the dict-type.